src/components/Editor/index.js

```diff
@@ -442,6 +442,7 @@ class Editor extends PureComponent<Props, State> {
     // to flash the previously saved selected line.
     if (
       line !== null &&
+      this.lastJumpLine !== 1 &&
```

Comment:
the new work drops the notion of `lastJumpLine` and perhaps fixes the problem :)

not sure

Reply:
@jasonLaster  First impression: it's a lot faster than the old one, yummy! Getting to work.

Reply:
awesome!

Reply:
@jasonLaster We can completely remove this pr, the issue is no more. There is however another bug introduced. If you scroll a document, reload the debugger, the document will reload on line 1 column 1 instead of on the scroll state it was before.

Reply:
hmm - i wonder how that is new... we don't really store the scroll state and we'll probably remove the source while it is loading

Reply:
Also another bug, if you open a document after a prettified source has been created and scrolled, the new document opens scrolled.
![pretty-scroll](https://user-images.githubusercontent.com/23530054/34300340-db1092ca-e727-11e7-89d1-8744d2d6db86.gif)